ios/Gemfile.lock

```diff
@@ -283,22 +299,21 @@ GEM
       colored2 (~> 3.1)
       nanaimo (~> 0.3.0)
       rexml (>= 3.3.6, < 4.0)
-    xcpretty (0.3.0)
-      rouge (~> 2.0.7)
+    xcpretty (0.4.1)
+      rouge (~> 3.28.0)
     xcpretty-travis-formatter (1.0.1)
       xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
-  arm64-darwin-21
-  arm64-darwin-22
   ruby
 
 DEPENDENCIES
   activesupport (>= 6.1.7.5, != 7.1.0)
   cocoapods (>= 1.13, != 1.15.1, != 1.15.0)
+  concurrent-ruby (< 1.3.4)
   fastlane
   fastlane-plugin-browserstack
   xcodeproj (< 1.26.0)
 
 BUNDLED WITH
-   2.4.7
+   2.1.4
```

Comment:
:question: can you explain why we have to downgrade that much?

Reply:
That's a good question, thanks, I'll see if I can change that.